tests/integration/targets/aws_secret/tasks/replication.yml

```diff
@@ -0,0 +1,64 @@
+---
+- block:
+  # ============================================================
+  # Creation/Deletion testing
+  # ============================================================
+  - name: add secret to AWS Secrets Manager
+    aws_secret:
```

Comment:
I'd like to see a test that is also using a kms key id.

Reply:
Is there a known kms key in the AWS CI environment?  Otherwise I can use `alias/aws/secretsmanager`